src/commands/setting/setShop.js

```diff
@@ -1086,7 +1086,8 @@ module.exports = {
         const selectMenuActionRow = new ActionRowBuilder().addComponents(selectMenu)
         const buttonActionRow = new ActionRowBuilder().addComponents(doneCancelButton)
 
-        const guide = await reply.send(locale(`SETSHOP.EDIT_GUIDE`), {
+        let guide = null
```

Comment:
don't mind my first review lmao; i just read the code again. i was thinking of doing it this way for my predictable flow. so nothing mutated.
```javascript
const guideBase = await reply.send(locale(`SETSHOP.EDIT_GUIDE`), { ... });
const guide = isInteractionCallbackResponse(guideBase) ? guideBase.resource.message : guideBase;
```